### PR TITLE
Adding testing scenarios

### PR DESCRIPTION
**Pull request by EricJohnson327**
## Summary of the pull request
Adding testing scenarios for manual validation before shipping updates

## PR checklist
- [ ] Closes #xxx
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

File: tools/ExtensionLibrary/DevHome.ExtensionLibrary.UnitTest/TestingScenarios/ExtensionLibrary.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Extension Library Tests
+If your code affects the extension library page, please manually verify these scenarios.
+
+## Scenarios
+Please make sure to verify all these scenarios.
+
+1. User can install/uninstall an extension
+1. User can enable/disable an extension
```
Comment (krschau) [inline tools/ExtensionLibrary/DevHome.ExtensionLibrary.UnitTest/TestingScenarios/ExtensionLibrary.md:8]:
```suggestion
1. User can enable/disable an extension
1. User can click through to extension's settings page if there is one
```

File: tools/SetupFlow/DevHome.SetupFlow.UnitTest/TestingScenarios/AppInstall.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# App Install Tests
+If your code affects app install in Machine Configuration, please manually verify these scenarios.
+
+## Scenarios
+Please make sure to verify all these scenarios.
+
+1. User can search for applications in search bar
+1. User can install applications
+1. User can not install an application that is already installed
```
Comment (krschau) [inline tools/SetupFlow/DevHome.SetupFlow.UnitTest/TestingScenarios/AppInstall.md:9]:
```suggestion
1. User cannot install an application that is already installed 
```

File: tools/SetupFlow/DevHome.SetupFlow.UnitTest/TestingScenarios/Repository.md [hunk 3]
```diff
@@ -18,9 +20,12 @@ Please make sure to verify all these scenarios.
...
 
-## Providers
-### One enabled provider
+### Providers
+#### One enabled provider
 1. If the user is logged in to the repository provider, clicking on the "Account" tab brings the user to the list of repos they have access to.
 
-### Multiple enabled providers
+#### Multiple enabled providers
 1. The account tab displays a list of all enabled providers the user can choose.  The names displayed is the long form.  Example "Dev Home GitHub Extension (dev)" and not "Github"
```
Comment (krschau) [inline tools/SetupFlow/DevHome.SetupFlow.UnitTest/TestingScenarios/Repository.md:28]:
```suggestion
1. The account tab displays a list of all enabled providers the user can choose.  The names displayed are the long form.  Example "Dev Home GitHub Extension (Dev)" and not "GitHub".
```

File: test/TestingScenarios/DevHome.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# Dev Home Core Tests
+Please manually verify these scenarios.  This is required for all PRs that affect Dev Home.
+
+## Scenarios
+Please make sure to verify all these scenarios.
+
+1. User can install Dev Home
+1. User can click all 'learn more' links on the Introducing Dev Home page
+1. User can click on “Connect accounts” button in the “Introducing Dev Home” page to navigate to Settings-> Accounts page
```
Comment (krschau) [inline test/TestingScenarios/DevHome.md:9]:
```suggestion
1. On first launch from new installation, user is brought to the "Introducing Dev Home" page

### Introducing Dev Home page
1. User can click "Get started" to be brought to the Machine configuration page
1. User can click "Explore extensions" to be brought to the Extensions page
1. User can click "Pin widgets" to be brought to the Dashboard page
1. User can click “Connect accounts” to be brought to Settings -> Accounts page
1. User can click all "learn more" links on the Introducing Dev Home page
1. User can resize the window and card sizes are responsive
```

File: tools/Dashboard/DevHome.Dashboard.UnitTest/TestingScenarios/Dashboard.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Dashboard/Widgets Tests
+If your code affects the dashboard or widgets, please manually verify these scenarios.
+
+## Scenarios
+Please make sure to verify all these scenarios.
+
+1. Add system widget and persists after restart
+1. Add extension widget and persists after restart
```
Comment (krschau) [inline tools/Dashboard/DevHome.Dashboard.UnitTest/TestingScenarios/Dashboard.md:8]:
```suggestion
1. Add system widget and ensure it persists after Dev Home restart
1. Add extension widget ensure it persists after Dev Home restart
```

File: tools/Dashboard/DevHome.Dashboard.UnitTest/TestingScenarios/Dashboard.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Dashboard/Widgets Tests
+If your code affects the dashboard or widgets, please manually verify these scenarios.
+
+## Scenarios
+Please make sure to verify all these scenarios.
+
+1. Add system widget and persists after restart
+1. Add extension widget and persists after restart
+1. Rearrange widgets
```
Comment (krschau) [inline tools/Dashboard/DevHome.Dashboard.UnitTest/TestingScenarios/Dashboard.md:9]:
```suggestion
1. Rearrange widgets via drag and drop
```

File: tools/Dashboard/DevHome.Dashboard.UnitTest/TestingScenarios/Dashboard.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+If your code affects the dashboard or widgets, please manually verify these scenarios.
+
+## Scenarios
+Please make sure to verify all these scenarios.
+
+1. Add system widget and persists after restart
+1. Add extension widget and persists after restart
+1. Rearrange widgets
+1. System widgets can be customized and persists after restart
+1. Extension widgets can be customized and persists after restart
```
Comment (krschau) [inline tools/Dashboard/DevHome.Dashboard.UnitTest/TestingScenarios/Dashboard.md:11]:
```suggestion
1. System widgets can be customized and persist after restart
1. Extension widgets can be customized and persist after restart
```

**Review by krschau — COMMENTED** [review]
I got *real* nitpicky but all with suggestions so you can easily just batch apply them if you want.

**Review by krschau — APPROVED** [review]

**Review by vineeththomasalex — APPROVED** [review]

**Pull request merged**